sdk/typescript/src/types/transactions.ts

```diff
@@ -4,15 +4,15 @@
 import { ObjectOwner, SuiAddress, TransactionDigest } from './common';
 import { SuiMovePackage, SuiObject, SuiObjectRef } from './objects';
 
-export type TransferCoin = {
+export type PublicTransferObject = {
```

Comment:
Thanks for making the change Todd!

cc @666lcz looks like we can leverage this for NFT as well. we will need to change the `signer & provider` accordingly, probably also `sui-objects/Coin.ts` before migrating it to SDK or a shared lib, right?

Reply:
yes, we also need to update https://github.com/MystenLabs/sui/blob/8a3d0789caecb5751036a2b308a9c64f600c7cbe/sdk/typescript/src/signers/signer-with-provider.ts#L71 , but I can help make this change